src/olympia/devhub/templates/devhub/new-landing/components/navigation.html

```diff
@@ -1,49 +1,49 @@
-<nav class="DevHub-Navigation content scheme-light">
+<nav id="nav-menu" class="DevHub-Navigation content scheme-light">
   <div class="DevHub-Navigation-list-wrapper">
     <ul>
-      <li>
-        <a href="{{ url('devhub.index') }}" class="Logo">
-          <span>{{ _('Add-on Developer Hub') }}</span>
-        </a>
+      <li class="DevHub-Navigation-Logo-And-Close">
+        <div class="DevHub-Navigation-Logo">
+          <a href="{{ url('devhub.index') }}" class="Logo">
+            <span>{{ _('Add-on Developer Hub') }}</span>
+          </a>
+        </div>
+        <div class="DevHub-Navigation-Close"><a href="#">&#10006;</a></div>
       </li>
 
       {% if user_authenticated %}
-        <li class="show-on-desktop"><a href="{{ url('devhub.addons') }}">
+        <li class="in-hamburger-menu"><a href="{{ url('devhub.addons') }}">
           {{ _('My Add-ons') }}
         </a></li>
       {% endif %}
 
-      <li class="show-on-desktop"><a href="https://extensionworkshop.com/?utm_source=addons.mozilla.org&utm_medium=referral&utm_content=devhub-menu-link">
+      <li class="in-hamburger-menu"><a href="https://extensionworkshop.com/?utm_source=addons.mozilla.org&utm_medium=referral&utm_content=devhub-menu-link">
```

Comment:
Can we have URLs by environment (-dev, stage, prod)?

Reply:
We generally link to the public version of all external websites (e.g devhub) even on addons-dev, so I'm not sure about this.  We _do_ link to extensionworkshop a lot though, so https://github.com/mozilla/addons-server/pull/12703/commits/0bb8c99971435bc172e1306a9d8456825795e222 adds the _ability_ to change all these with single a setting.  Ops can set the variable to have addons-stage or addons-dev link to the extensionsworkshop dev/stage site if that's wanted (or with a followup commit it can be added to the relevant settings files in a single line)

Reply:
Oh I see, ok, thanks :)